ykrt/src/compile/jitc_yk/codegen/x64/deopt.rs

```diff
@@ -41,8 +41,8 @@ pub(crate) extern "C" fn __yk_guardcheck(gidx: u64) -> *const libc::c_void {
     let info = &ctr.deoptinfo[&usize::from(gidx)];
     if let Some(st) = info.guard.ctr() {
         let staddr = st.entry();
-        let mt = Arc::clone(&ctr.mt);
-        mt.log.log(Verbosity::JITEvent, "execute-side-trace");
+        //let mt = Arc::clone(&ctr.mt);
```

Comment:
Eh?

Reply:
Oh, I wanted to `git absorb` this but I must have commited it accidentally. Yeah, it's put back in https://github.com/ykjit/yk/pull/1422/commits/0a3d0820e5bc843655f9e6236b5a82d2b27dbf83. Sorry.